test/module/irohad/validation/chain_validation_test.cpp

```diff
@@ -37,19 +43,61 @@ class ChainValidationTest : public ::testing::Test {
     storage = std::make_shared<MockMutableStorage>();
     query = std::make_shared<MockWsvQuery>();
 
-    peer.pubkey.fill(2);
+    // TODO: 14-02-2018 Alexey Chernyshov remove after replacement
+    // with shared_model https://soramitsu.atlassian.net/browse/IR-903
+    std::copy(
+        public_key.blob().begin(), public_key.blob().end(), peer.pubkey.data());
     peers = std::vector<Peer>{peer};
+  }
+
+  /**
+   * Get block builder to build blocks for tests
+   * @return block builder
+   */
+  auto getBlockBuilder() const {
+    constexpr auto kTotal = (1 << 5) - 1;
+    return shared_model::proto::TemplateBlockBuilder<
+               kTotal,
+               shared_model::validation::DefaultBlockValidator,
+               shared_model::proto::Block>()
+        .txNumber(0)
+        .height(1)
+        .prevHash(hash)
+        .createdTime(iroha::time::now());
+  }
 
-    block.sigs.emplace_back();
-    block.sigs.back().pubkey = peer.pubkey;
-    block.prev_hash.fill(0);
-    hash = block.prev_hash;
+  /**
+   * Add signature to the block.
+   */
+  void addSignature(shared_model::interface::Block &block,
+                    shared_model::interface::types::PubkeyType &pubkey) {
+    shared_model::builder::SignatureBuilder<
+        shared_model::proto::SignatureBuilder,
+        shared_model::validation::FieldValidator>
+        builder;
+    auto signature = builder.publicKey(pubkey).build();
+
+    signature.match(
+        [&](shared_model::builder::BuilderResult<
+            shared_model::interface::Signature>::ValueType &sig) {
+          block.addSignature(sig.value);
+        },
+        [](shared_model::builder::BuilderResult<
+            shared_model::interface::Signature>::ErrorType &e) {
+          FAIL() << *e.error;
+        });
   }
 
+  shared_model::interface::types::PubkeyType public_key =
+      shared_model::interface::types::PubkeyType(std::string(32, '2'));
+
+  // TODO: 14-02-2018 Alexey Chernyshov remove after replacement
+  // with shared_model https://soramitsu.atlassian.net/browse/IR-903
   Peer peer;
   std::vector<Peer> peers;
-  Block block;
-  hash256_t hash;
+
+  shared_model::crypto::Hash hash =
```

Comment:
in class member initialisation we can use brace-or-equal-initializer. Otherwise there is ambiguity with function declaration. I don't know the difference, for me equal initialisation looks more familiar.
```
class A {
  int a = 1;
  int b{2};
  // int c(3); // forbidden
};
```